### PR TITLE
update doc

### PR DESCRIPTION
**Pull request by JBlond**

### CODE REVIEW

**Comment by DigiLive** [comment]
I will review the changes tommorow as I was working on inline CLI diff today.

File: lib/jblond/Diff.php [hunk 1]
```diff
@@ -50,8 +50,9 @@ class Diff
     private $groupedCodes;
 
     /**
-     * @var array   Associative array containing the default options available for the diff class and their default
-     *              value.
+     * @var array<string> Associative array containing the default options available
```
Comment (DigiLive) [inline lib/jblond/Diff.php:53]: It's a new syntax I never use, but shouldn't it be `array<string, string>` in this case?

File: lib/jblond/Diff.php [hunk 2]
```diff
@@ -111,8 +112,8 @@ public function __construct($version1, $version2, array $options = [])
      *
      * @param mixed $var Variable to get type from.
      *
-     * @return int  Number indicating the type of the variable. 0 for array type and 1 for string type.
-     * @throws InvalidArgumentException    When the type isn't 'array' or 'string'.
+     * @return int umber indicating the type of the variable. 0 for array type and 1 for string type.
```
Comment (DigiLive) [inline lib/jblond/Diff.php:115]: Typo here...

File: lib/jblond/Diff.php (hunk 1 above)
Comment (JBlond) [reply lib/jblond/Diff.php:53]: That is the syntax for an array of strings. other example would be array<string|array>
__

File: lib/jblond/Diff.php (hunk 2 above)
Comment (JBlond) [reply lib/jblond/Diff.php]: I corrected that, Well spotted

File: lib/jblond/Diff.php (hunk 1 above)
Comment (DigiLive) [reply lib/jblond/Diff.php:53]: I've got my info from https://suckup.de/2020/02/modern-phpdoc-annotations/
It's all fine with me as long as it's compatible with PhpStorm 😇

**Pull request merged**